src/applications/facility-locator/components/ServiceTypeAhead.jsx

```diff
@@ -42,27 +42,44 @@ class ServiceTypeAhead extends Component {
 
   optionClasses = selected => classNames('dropdown-option', { selected });
 
-  shouldShow = (input, svc) =>
-    input.length >= 2 &&
-    svc &&
-    svc.name &&
-    svc.name
-      .trim()
-      .toLowerCase()
-      .includes(input.toLowerCase());
+  getSpecialtyName = specialty => {
+    if (!specialty) return null;
+
+    const specialization = specialty.specialization
+      ? ` - ${specialty.specialization}`
+      : '';
+
+    return specialty.classification + specialization;
+  };
```

Comment:
This can be removed once https://github.com/department-of-veterans-affairs/vets-api/pull/4647 has been merged.